README.md

```diff
@@ -1,12 +1,12 @@
-# Plugin Installation for IntelliJ and Android Studio
+# Plugin Installation for IntelliJ, Android Studio and Xcode
 
-Follow the steps below to install the plugin for IntelliJ and Android Studio.
+Follow the steps below to install the plugin.
 
 ## Step 1: Install the Flow authentication agent
 
 The first step is to install the agent that will connect to the Flow portal for authentication.
 
-## Linux and macOS
+### Linux and macOS
 
 To do this, execute the following command in your terminal:
```

Comment:
com a adição do xcode, eu mudaria aqui:
"For IntelliJ and Android Studio, execute the following command in your terminal:"